parse-nessus-xml.py

```diff
@@ -49,24 +66,46 @@
             "plugin_output": plugin_output,
             "hosts": hosts
         }
+
+        # Match on known daemons:
+        if plugin_id == 33851:
+            for line in plugin_output.splitlines():
+                if (len(line) < 1):
+                    continue
+                if "The following running daemons are not managed by dpkg" in line:
+                    continue
+                if (line == "/bin/kube2iam"):
+                    daemon_count += 1
+                    continue
+                if re.search(rf'/var/vcap/bosh/bin/(bosh-agent|monit)', line):
+                    daemon_count += 1
+                    continue
+                if re.search(rf'^/var/vcap/data/packages/({DAEMONS})/[0-9a-f]+/bin/{DAEMONS}$', line):
```

Comment:
maybe over-optimization for this script, but if you compile the regex outside the loop, this will run significantly faster

Reply:
It runs fast enough, so I'm going to keep the RE right where it's used.